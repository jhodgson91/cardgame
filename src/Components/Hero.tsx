import React, { FC } from 'react';
import './Main.scss'

export interface Props {
  title: string;
}

const Hero: FC<Props> = ({ title }: Props) => (
  <section id="hero">
    <div className="grid-container">
      <div className="grid-x grid-padding-x align-center text-center">
        <div className="small-12">
          <h1>Card game - {title}</h1>
        </div>
      </div>
    </div>
  </section>
)

export default Hero
