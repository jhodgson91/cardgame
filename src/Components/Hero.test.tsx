import * as React from 'react';
import * as enzyme from 'enzyme';
import Hero from './Hero';

it('renders the correct text', () => {
  const hero = enzyme.shallow(<Hero title='test' />);
  expect(hero.find("h1").text()).toBe('Card game - test')
});