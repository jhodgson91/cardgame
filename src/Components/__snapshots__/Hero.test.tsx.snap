// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same as last time with a button 1`] = `
<section
  id="hero"
>
  <div
    className="grid-container"
  >
    <div
      className="grid-x grid-padding-x align-center text-center"
    >
      <div
        className="small-12"
      >
        <h1>
          Card game - 
          Test
        </h1>
      </div>
    </div>
  </div>
</section>
`;
